pkg/spdk/engine.go

```diff
@@ -49,6 +49,7 @@ type Engine struct {
 	Nqn                string
 	Nguid              string
 
+	initiator    *nvme.Initiator
```

Comment:
Is caching the `initiator` just to avoid reloading the device?

Reply:
Yes. Launching an `initiator` is trivial. And it's frequently used in `engine`